keps/sig-node/1948-add-deallocate-to-device-plugin-api/kep.yaml

```
@@ -0,0 +1,27 @@
+title: Add Deallocate and PostStopContainer to device plugin API
+kep-number: 1948
+authors:
+  - "@mewais"
+owning-sig: sig-node
+participating-sigs: []
+status: implementable
+creation-date: 2020-08-18
+
+reviewers:
+  - TBD
+approvers:
+  - TBD
+prr-approvers:
+  - "@dchen1107"
+see-also: []
+replaces: []
+
+# The target maturity stage in the current dev cycle for this KEP.
+stage: alpha
+
+# The milestone at which this feature was, or is targeted to be, at each stage.
+latest-milestone: "v1.21"
+milestone:
+  alpha: "v1.21"
```

these milestones are stale now please update

---

How so? Isn't the latest version 1.20.4?!

---

Oh my mistake, I didn't see the new 1.21 alphas.

---

```suggestion
  alpha: "v1.22"
```

and else where accordingly, since v1.21 is pre-released. (https://github.com/kubernetes/kubernetes/releases)